src/api/app/models/package.rb

```diff
@@ -747,9 +747,9 @@ def delete_on_backend
         # ignore this error, backend was out of sync
         logger.warn("Package #{self.project.name}/#{self.name} was already missing on backend on removal")
       end
-      logger.tagged('backend_sync') { logger.debug "Deleted Package #{self.project.name}/#{self.name}" }
+      logger.tagged('backend_sync') { logger.warn "Deleted Package #{self.project.name}/#{self.name}" }
     else
-      if @commit_opts[:no_backend_write]
+      if @commit_opts[:no_backend_write] && !@commit_opts[:project_destroy_transaction]
```

Comment:
Hm, but this would still be logged as warning in the else branch. What am I missing? So with the change it would always log it with "global_write_through is off". Is that correct?